registry/docker-compose.yaml

```diff
@@ -1,17 +1,41 @@
+version: "3"
+
+networks:
+  gitea:
+    external: false
+
+volumes:
+  gitea:
+    driver: local
+
 services:
-  registry:
+  server:
+    image: gitea/gitea:1.18.5
```

Comment:
We could at least use the latest 1.19. 😜

Reply:
1.19 was not out when I did this. I'll update it!